playbook

```diff
@@ -0,0 +1,65 @@
+---
+- name: Install To-Do App on new web server
+  hosts: managed_node
+  remote_user: ec2-user
+  vars:
+    secret_key: ffca2923fb0c9d9ebd47daf91d7e7065
+    secret_token: b98402aff272a5bc83ed04e819a196de32846a18c59f9bb6aec3b16c082a2062
+    secret_secret: 9b659fc10bb91b33c059890dab8fe6fa222c9797e952cf38093e15ba9bfd7aaa
```

Comment:
This includes your real Trello key and token, which are now leaked publicly! We should rotate your token at least, which [can be done from here](https://trello.com/my/account) by revoking the Token listed under "Applications" and then [creating a new one](https://trello.com/app-key)

In future, we should ask the user to input any sensitive values to avoid committing them into our code, for example [using the `vars_prompt` section](https://docs.ansible.com/ansible/latest/user_guide/playbooks_prompts.html)